app/models/mapping_rule.rb

```diff
@@ -18,6 +18,36 @@ def legacy_as_json(mapping_rules, version:)
       }
     end
 
+    def as_txt(mappings, version:)
+      header = <<~HEADER
+        # Shopify Product Taxonomy - Mappings: #{version}
+        # Format:
+        # {input_taxonomy_version} -> {output_taxonomy_version} # Mappings A start
```

Comment:
Should we add some `grepable` phrase/marker for these? Should we make a dist file _per mapping_ instead of a mega one 🤔